packages/worker/src/worker.ts

```diff
@@ -90,6 +90,7 @@ import { workflowLogAttributes } from './workflow-log-interceptor';
 import { defaultWorflowInterceptorModules, WorkflowCodeBundler } from './workflow/bundler';
 import { Workflow, WorkflowCreator } from './workflow/interface';
 import { ThreadedVMWorkflowCreator } from './workflow/threaded-vm';
+import { ReusableVMWorkflowCreator as ReusableVMWorkflowCreator } from './workflow/reusable-vm';
```

Comment:
nit: Dont alias to same name

Reply:
Probably due to somre refactor, will fix.